backend/data/src/main/kotlin/io/tolgee/configuration/tolgee/S3Settings.kt

```diff
@@ -40,4 +40,6 @@ class S3Settings(
   override var signingRegion: String? = null,
   @DocProperty(description = "Name of the bucket where Tolgee will store its files.")
   override var bucketName: String? = null,
+  @DocProperty(description = "Optional subfolder structure within s3 bucket to which content will be stored")
```

Comment:
I would add an example here, eg. to show if heading slash or trailing slash is ok.